scripts/test-many-configs.sh

```diff
@@ -0,0 +1,419 @@
+        else
+            cd ${projName}
+        fi
+        if [ -d "detection-results" ]; then
+            echo "Success: dt.cache.absolute.path configuration works as intended."
+            return 0
+        fi
+    fi
+    echo "Error: dt.cache.absolute.path not working as intended. %%%%%"
+    flag=1
+    return 1
+}
+
+
+
+function flakyTestsFound() {
+    testNum=$1
+    expectedTests=$2
+    currModule=$3
+    if [[ $expectedTests == -* ]]; then
+        expectedTests=$(echo ${expectedTests} | cut -d'-' -f2)
+        if grep "$expectedTests" ${testNum}.log; then
+            echo "Confirmed Expected Project Failure in ${testNum}"
+            return 0
+        else
+            echo "ERROR: Unexpected Project Failure in ${testNum} %%%%%"
+            flag=1
+            return 1
+        fi
+    else
+        if [[ ${currModule} != "" ]]; then
+            cd ${currModule}
+        fi
+        cd ${testNum}
+        cd detection-results
+        numFlakyTests=$(awk '1' list.txt | wc -l)
+        cd ${projectDirectory}
+
+        if [[ ${expectedTests} == ${numFlakyTests} ]]; then
+            echo "All expected tests were found in ${testNum}."
+            return 0
+        else
+            if [ $expectedTests -gt $numFlakyTests ]; then
+                let "x = $expectedTests - $numFlakyTests"
+                echo "There were $x fewer tests found than expected in ${testNum}. %%%%%"
+                flag=1
+                return 1
+            else
+                let "x = $numFlakyTests - $expectedTests"
+                echo "There were $x more tests found than expected in ${testNum}. %%%%%"
+                flag=1
+                return 1
+           fi
+        fi
+    fi
+}
+
+
+
+function checkTimeout() {
+    testNum=$1
+    expectedTime=$2
+    if [[ ! -f ${testNum} ]]; then
+        return 0
+    fi
+
+    if grep "Using a timeout of ${expectedTime}" ${testNum}.log; then
+        echo "Success: ${testNum} used a timeout of ${expectedTime} as expected."
+        return 0
+    else
+        echo "Error: ${testNum} did not use a timeout. %%%%%"
+        flag=1
+        return 1
+    fi
+}
+
+
+
+
+while IFS="," read -r URL SHA MODULE testCount1 testCount2 testCount3 testCount4 testCount5 testCount6 testCount7 testCount8 testCount9; do
+
+
+    renamedRepo=${URL}"/"
+    readarray -d / -t starr <<< "${renamedRepo}"
+
+
+    #1. Clone the project.
+
+    if [[ ! -d ${starr[4]} ]]; then
+        git clone ${URL}.git ${starr[4]}
+    fi
+
+
+
+
+    #2. Navigate to proj and checkout SHA
+
+    cd ${starr[4]}
+    projectDirectory=$(pwd)
+    git checkout ${SHA}
+
+
+
+
+    #3. Modify pom file
+
+    git checkout -f .
+    bash ${scriptDir}/../pom-modify/modify-project.sh ${projectDirectory} 1.2.0-SNAPSHOT
+
+
+
+
+    #4 Maven install the proj
+
+    if [[ ${MODULE} != "" ]]; then
+        PL="-pl ${MODULE}"
+    else
+        PL=""
+    fi
+
+    if [[ $URL == "https://github.com/wildfly/wildfly" ]]; then
```

Comment:
Put a comment about how it is necessary to modify the repository location on this old version of wildfly.